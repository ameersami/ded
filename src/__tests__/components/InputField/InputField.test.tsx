import * as React from 'react';
import { shallow } from 'enzyme';

import Inputfield from '../../../components/InputField/InputField';

describe('<InputField/>', () => {
  test('should render', () => {
    const wrapper = shallow(<Inputfield/>);
    expect(wrapper).toMatchSnapshot();
  });
});
