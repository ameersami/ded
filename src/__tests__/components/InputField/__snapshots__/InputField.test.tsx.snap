// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputField/> should render 1`] = `
<styled.label
  className=""
>
  <styled.input
    disabled={false}
    onBlur={[Function]}
    onChange={[Function]}
    placeholder=" "
    type="text"
  />
  <styled.span>
    InputField
  </styled.span>
</styled.label>
`;
